src/lib/types/components/cart.d.ts

```
@@ -0,0 +1,61 @@
+    _id: string;
+    price: number;
+    quantity: number;
+    product: Product;
+}
+
+// cart data
+export interface CartData {
+    _id: string;
+    user: string;
+    cartItems: CartItem[];
+    appliedCoupons: any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)